easybuild/tools/github.py

```diff
@@ -2190,11 +2190,12 @@ def det_account_branch_for_pr(pr_id, github_user=None, pr_target_repo=None):
 
     # branch that corresponds with PR is supplied in form <account>:<branch_label>
     account = pr_data['head']['label'].split(':')[0]
+    repo = pr_data['head']['repo']['name']
     branch = ':'.join(pr_data['head']['label'].split(':')[1:])
     github_target = '%s/%s' % (pr_target_account, pr_target_repo)
     print_msg("Determined branch name corresponding to %s PR #%s: %s" % (github_target, pr_id, branch), log=_log)
 
-    return account, branch
+    return account, repo, branch
```

Comment:
Hmm, this is a breaking change to the API...

How about introducing a `det_account_repo_branch_for_pr` (or a more sensible name, like `det_pr_target_info`), and deprecating `det_account_branch_for_pr`?

Reply:
This is only used in the 2 places modified by this PR, and nowhere else i could find, might be overkill to leave the function unused only in case some dependecies is using it?

Worst case i would have the old function call this one and return only `account, branch`